### PR TITLE
[WIP] revamp the docs

### PR DESCRIPTION
**Pull request by martenson**
up to advanced dev topics, working on that now

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -2,34 +2,42 @@
 Installation
 ============
 
-The recommended approach for installing Planemo is to use Homebrew_ or
-linuxbrew_. To install planemo this way simply use the ``brew`` command as
-follows.
+The recommended approach for installing Planemo is to use Conda_
```
Comment (martenson) [inline docs/installation.rst]: I think @jmchilton was against this switch, I will revert.

**Comment by jmchilton** [comment]
Very nice so far! Thanks a million.

File: docs/installation.rst (hunk 1 above)
Comment (jmchilton) [inline docs/installation.rst]: Probably yes - That conda recipe doesn't get updated nearly as frequently as the Homebrew or PyPI recipe yet. I'm still nervous about using conda environments with Galaxy also - though I guess it should be fine.

I'll try to adjust my workflow with the GCC as impetus and hopefully we can reach a point where that conda recipe is my go to one.

**Comment by martenson** [comment]
We are back to Homebrew for planemo itself (but have Conda for seqtk and BWA)

**Comment by martenson** [comment]
The whole advanced tooldev topics section is now validated and concise. 

The only thing I am unclear now is whether we should add another tutorial introducing devs to Conda-controlled requirements?

**Comment by martenson** [comment]
fixed conflicts

**Comment by jmchilton** [comment]
I'm walking through the new docs on the new image and so far it is simply fantastic - this is a great improvement and I profoundly appreciate the effort you put into this! I can't thank you enough @martenson!

**Pull request merged**